wiki/input/cursor-visibility-and-catching.md

```diff
@@ -1,40 +1,138 @@
+		<td>Used for finer precision than the default arrow cursor</td>
+	</tr>
+	<tr style="cursor: pointer">
+		<td><code>Hand</code></td>
+		<td><img alt="pointer cursor" src="/assets/wiki/images/cursor-visibility-and-catching5.png" width="24" height="24"></td>
+		<td>Indicates hyperlink can be followed</td>
+	</tr>
+	<tr style="cursor: col-resize">
+		<td><code>HorizontalResize</code></td>
+		<td><img alt="ew-resize cursor" src="/assets/wiki/images/cursor-visibility-and-catching6.png" width="24" height="24"></td>
+		<td>Indicates item can be resized horizontally</td>
+	</tr>
+	<tr style="cursor: row-resize">
+		<td><code>VerticalResize</code></td>
+		<td><img alt="ns-resize cursor" src="/assets/wiki/images/cursor-visibility-and-catching7.png" width="24" height="24"></td>
+		<td>Indicates item can be resized vertically</td>
+	</tr>
+	<tr style="cursor: nwse-resize">
+		<td><code>NWSEResize</code></td>
+		<td><img alt="nwse-resize cursor" src="/assets/wiki/images/cursor-visibility-and-catching8.png" width="24" height="24"></td>
+		<td>Indicates item corner can be resized inwards or outwards<br>
+		<strong>macOS:</strong> Uses private system API and may fail in future<br>
+		<strong>Linux:</strong> Uses newer standard that not all cursor themes support</td>
+	</tr>
+	<tr style="cursor: nesw-resize">
+		<td><code>NESWResize</code></td>
+		<td><img alt="nesw-resize cursor" src="/assets/wiki/images/cursor-visibility-and-catching9.png" width="24" height="24"></td>
+		<td>Indicates item corner can be resized inwards or outwards<br>
+		<strong>macOS:</strong> Uses private system API and may fail in future<br>
+		<strong>Linux:</strong> Uses newer standard that not all cursor themes support</td>
+	</tr>
+	<tr style="cursor: all-scroll">
+		<td><code>AllResize</code></td>
+		<td><img alt="all-scroll cursor" src="/assets/wiki/images/cursor-visibility-and-catching10.png" width="24" height="24"></td>
+		<td>Indicates the ability to scroll/pan in all directions</td>
+	</tr>
+	<tr style="cursor: not-allowed">
+		<td><code>NotAllowed</code></td>
+		<td><img alt="not-allowed cursor" src="/assets/wiki/images/cursor-visibility-and-catching11.png" width="24" height="24"></td>
+		<td>Indicates an action is prohibited<br>
+		<strong>Linux:</strong> Uses newer standard that not all cursor themes support</td>
+	</tr>
+</table>
+
+Note that `NWSEResize` onwards are new in libGDX 1.10.1-SNAPSHOT. They aren't present in earlier versions.
+
+## Additional resources
+
+If you wish to let your HTML5 game use system cursors libGDX doesn't support, this is a good starting point:
+
+* [CSS `cursor` values](https://developer.mozilla.org/en-US/docs/Web/CSS/cursor#values)
+* [Interfacing with platform specific code](https://libgdx.com/wiki/app/interfacing-with-platform-specific-code)
+* [JSNI](http://www.gwtproject.org/doc/latest/DevGuideCodingBasicsJSNI.html)
+
+If you're willing to modify libGDX's source code, it is possible to properly hide the mouse cursor:
```

Comment:
Would you mind changing that to reflect https://github.com/libgdx/libgdx/pull/6822?

Reply:
Working on it. I've just noticed an issue with the cursors that I'm getting to the bottom of. If https://github.com/libgdx/libgdx/pull/6821 is merged I can also update it to reflect that, though at no comments I'm not holding my breath.

Reply:
Hm, so `move` is the same as `grabbing` in Firefox on my system. Fine on Chrome and fine on Windows. Probably something to do with https://github.com/ubuntu/yaru/issues/2880 so I won't note it.

Reply:
> If https://github.com/libgdx/libgdx/pull/6821 is merged I can also update it to reflect that, though at no comments I'm not holding my breath.

Android isn't really my jam, so we'll have to wait for someone else to review that.